data/test_config/config_with_comments_after_dumping.yml

```diff
@@ -0,0 +1,32 @@
+# here is some comment
+language: en
+
+# another comment
+pipeline:
+  - name: ConveRTTokenizer
+  - name: ConveRTFeaturizer
+  - name: RegexFeaturizer
+  - name: LexicalSyntacticFeaturizer
+  - name: CountVectorsFeaturizer
+  - name: CountVectorsFeaturizer
+    analyzer: "char_wb"
+    min_ngram: 1
+    max_ngram: 4
+  - name: DIETClassifier
+    epochs: 100
+  - name: EntitySynonymMapper
+  - name: ResponseSelector
+    epochs: 100
+# all the comments
+# so many
```

Comment:
https://tenor.com/view/dean-cain-comments-just-here-for-the-comments-gif-15260956